packages/@dcl/inspector/src/lib/data-layer/client/local-data-layer.ts

```diff
@@ -1,32 +1,12 @@
-import { createEngine } from '../host/engine'
-import { initRpcMethods } from '../host/rpc-methods'
-import { DataLayerRpcClient, DataLayerRpcServer, FileSystemInterface } from '../types'
-
-function wrapRpcClientFromRpcServer(server: DataLayerRpcServer): DataLayerRpcClient {
-  return server as any as DataLayerRpcClient
-}
+import { createDataLayerHost } from '../host'
+import { DataLayerRpcClient, FileSystemInterface } from '../types'
 
 /**
- * This RpcClient creates internally the server, implementing its own file system interface and engine.
+ * This RpcClient creates internally the DataLayer HOST, implementing its own file system interface and engine.
  * @param fs
  * @returns
  */
 export async function createLocalDataLayerRpcClient(fs: FileSystemInterface): Promise<DataLayerRpcClient> {
-  const engine = createEngine()
```

Comment:
This function only creates the `RpcClient`, the logic was moved to `host/index.ts` (and now it describes the DataLayer, and reused in the `sdk-commands`)